chain/types/actor_event.go

```diff
@@ -0,0 +1,72 @@
+package types
+
+import (
+	"github.com/ipfs/go-cid"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+)
+
+type ActorEventBlock struct {
```

Comment:
not now: in the future, I'd like to consider automatically translating cbor to json when requested by the user.